code/modules/clothing/glasses/hud_vr.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/clothing/glasses/omnihud
 	name = "\improper AR glasses"
-	desc = "The KHI-62 AR Glasses are a design from Kitsuhana Heavy Industries. These are a cheap export version \
+	desc = "The KHI-62 AR Glasses are a design from Vey Med. These are a cheap export version \
```

Comment:
I would suggest changing KHI to another abbreviation like 'VM'. KHI as an acronym also means nothing in our lore.

Reply:
Same goes for the KHI in the other glasses you modified.

Reply:
oh never even thought of that, will do

Reply:
done btw